Cargo.toml

```
@@ -18,6 +18,7 @@ readme = "README.md"
 repository = "https://github.com/maciejhirsz/logos"
 rust-version = "1.74.0"
 version = "0.15.1"
+include = ["build.rs", "/src", "LICENSE-MIT", "LICENSE-APACHE"]
```

```suggestion
include = ["build.rs", "src/**", "LICENSE-MIT", "LICENSE-APACHE"]
```

I think it should be `src/**`, per https://doc.rust-lang.org/cargo/reference/manifest.html#the-exclude-and-include-fields

---

Which do you prefer? (I personally prefer `/src` or `/src/`, but I'm happy to change it to something else too ^^)